webapp/components/drawer/Drawer.js

```diff
@@ -123,27 +160,38 @@ export default ({
         widget={widget}
         disabled={widget.disabled}
         expanded={expand}
-        classes={classes}
+        classes={drawerClasses}
         createOrFocusWidget={createOrFocusWidget}
         status={status}
       />
     );
   };
-
+  const paperClasses = `${expand ? drawerClasses.openDrawer : drawerClasses.closeDrawer} ${classes.root}`;
   return (
-    <Paper elevation={0} className={expand ? classes.openDrawer : classes.closeDrawer}>
-      <div className={classes.container}>
+    <Paper elevation={0} className={paperClasses}>
+      <div className={drawerClasses.container}>
         <Box p={2}>
+          <Box className="drawerListBox">
+            <Typography variant="body2">Model Specification</Typography>
```

Comment:
If you're in the Model > Explore view, this heading should't be "Model Specification" since we show here the Plots of the model, so rather "Analysis" or "Plots". Which of both is not so important for now